src/components/Support/WaysToSupport/index.tsx

```diff
@@ -4,32 +4,32 @@ import CommonCard from "@/components/Common/CommonCard"
 const WaysToSupport = () => {
     const data = [
         {
-            title: "How to contribute",
+            title: "Contribute to Adoptium Projects",
             description:
-                "Contribute to our projects with technical and non-technical development",
+                "Contribute to our projects with technical and non-technical development.",
             button: "How to contribute",
             href: "/contributing/",
         },
         {
             title: "Become a sustainer",
-            description: "Join us in strengthening the future of Eclipse Temurin",
+            description: "Help ensure Temurin remains a trusted, high-performance runtime for developers everywhere.",
```

Comment:
```suggestion
            description: "Help ensure Temurin remains a trusted, high-performance runtime for users everywhere.",
```

Not just for developers ;-)